docs/python/generators/doxygen.py

```
@@ -263,20 +312,26 @@ def _example_priority(e):
+                                if re.search(r"\b" + re.escape(n) + r"\s*\(", snippet):
+                                    return 0
+                            # header include for the documented name's class
+                            for n in names_to_search:
+                                if not n:
+                                    continue
+                                if re.search(r"#include\s+[<\"]AUI/.+/" + re.escape(n) + r"\.h[>\"]", text):
+                                    return 1
+                            # unquoted occurrence of documented name on nontrivial line
                             try:
                                 from docs.python.generators.examples_helpers import _find_unquoted_word_on_nontrivial_line
                                 if any(_find_unquoted_word_on_nontrivial_line(n, text) for n in names_to_search if n):
```

Ruff (0.15.21)

[warning] Function definition does not bind loop variable `names_to_search`

(B023)